operations.py

```diff
@@ -67,5 +68,41 @@ def mat_subtract(self, matrix1, matrix2):
             logging.error("Unexpected error:", e)
         return constants.ERROR
 
+    def print_char_count(self, variable):
+        try:
+            if isinstance(variable, Future):
+                variable = self.workerObj.get_result_from_worker(variable)
+            if isinstance(variable, bytes):
+                variable = variable.decode('utf-8')
+            if not isinstance(variable, str):
```

Comment:
Variable here can never be string here. If a string is passed to taskData, there will be a throw in `to_proto()` in task.py.
Please check the flow once and remove if not necessary.

Reply:
Variable can become string after 75-76

Reply:
Won't line 76 thow if it's not decodable to string?